docs/cli.md

```diff
@@ -12,13 +12,15 @@ NAME:
    config-server-encrypt-value - Encrypt a string using a Spring Cloud Services configuration server
 
 USAGE:
-      cf config-server-encrypt-value CONFIG_SERVER_INSTANCE_NAME VALUE_TO_ENCRYPT
+      cf config-server-encrypt-value CONFIG_SERVER_INSTANCE_NAME [VALUE_TO_ENCRYPT]
 
-Note: if VALUE_TO_ENCRYPT contains characters that are special to the shell, it may be necessary
-to use escape characters or quotes. Refer to the shell's man page for details.
+      NOTE: VALUE_TO_ENCRYPT is optional if --file-to-encrypt flag is provided.
```

Comment:
What does it mean if the value is provided in the file case?

I think it would be better to require that `VALUE_TO_ENCRYPT` must be omitted if the `--file-to-encrypt` flag is specified.

Reply:
It will ignore the value if `-f` is given. I documented that somewhere (just below the snippet you show here). I thought about making them mutually exclusive and failing if both are given, but quite honestly...it was getting late in the day and I'm on the road today (and will soon have spotty wifi), so I decided to go with what would get something checked in at the end of the day.

Wifi-permitting, I'll take another look and see about making them mutually exclusive.

Reply:
Thanks. It's always worth making user-facing interfaces as bullet-proof as possible.